### ISSUE TITLE
Is it possible to add Canada Computers to the list of supported stores. They are one of the big retailers in Canada

### ISSUE THREAD

**Issue by tempestbk** [jef/streetmerchant#730]

**Comment by omelettedufromagee** [comment]
Hi, I added memoryexpress. I'm currently working on CC.

**Comment by tempestbk** [comment]
Thank you Thank you Thank you

I'm new to this whole thing. Is it a lot of man hours?

Dan

**Comment by omelettedufromagee** [comment]
Few hours.

**Comment by bassman2112** [comment]
I've added Ryzen 5000 series and EVGA 3000 series with this commit, however it needs `labels.maxPrice` and `labels.outOfStock` information added.

**Comment by omelettedufromagee** [comment]
That's cool. Thanks!

**Comment by applesauce9** [comment]
I believe the way the canada computers site works, it shows as in stock when the page loads, then the stock section gets updated after. Is it possible that `waitUntil: 'domcontentloaded` doesn't wait long enough? I get a lot of false positives.

**Comment by rodrigorodrigo** [comment]
> 
> 
> I believe the way the canada computers site works, it shows as in stock when the page loads, then the stock section gets updated after. Is it possible that `waitUntil: 'domcontentloaded` doesn't wait long enough? I get a lot of false positives.

I am currently testing this store and only got two false positives in the last four hours. The screenshots make me believe it had to do with their webserver not redirecting fast enough. Both were in the same GPU so it may be the link. I will be investigating.

**Comment by applesauce9** [comment]
Ahh I should turn screenshots back on and check. Thanks

**Comment by jef** [comment]
`waitUntil` should usually default to `networkIdle2`. We can remove this if it's bringing more success. Do you mind testing and removing that line of code from the `canadacomputers.ts`?

**Comment by applesauce9** [comment]
I tried it with networkidle0 and got timeouts. Tried it with networkidle2 and get the false positives again. The screenshots that I now turned back on show that it's going to a language selection screen on the canadacomputers website or sometimes their facebook page.

![image](https://user-images.githubusercontent.com/74373281/99082623-65e6a780-2592-11eb-9a18-790ecf15b2f2.png)
![image](https://user-images.githubusercontent.com/74373281/99082641-6b43f200-2592-11eb-917a-36280e111d37.png)
